packages/flutter/lib/src/widgets/raw_keyboard_listener.dart

```diff
@@ -14,28 +14,35 @@ export 'package:flutter/services.dart' show RawKeyEvent;
 /// A widget that calls a callback whenever the user presses or releases a key
 /// on a keyboard.
 ///
+/// The [RawKeyboardListener] is deprecated and will be removed. Use
+/// [KeyboardListener] instead.
+///
 /// A [RawKeyboardListener] is useful for listening to raw key events and
 /// hardware buttons that are represented as keys. Typically used by games and
 /// other apps that use keyboards for purposes other than text entry.
 ///
 /// For text entry, consider using a [EditableText], which integrates with
 /// on-screen keyboards and input method editors (IMEs).
 ///
-/// The [RawKeyboardListener] is different from [KeyboardListener] in that
-/// [RawKeyboardListener] uses the legacy [RawKeyboard] API. Use
-/// [KeyboardListener] if possible.
-///
 /// See also:
 ///
-///  * [EditableText], which should be used instead of this widget for text
-///    entry.
-///  * [KeyboardListener], a similar widget based on the newer
-///    [HardwareKeyboard] API.
+/// * [EditableText], which should be used instead of this widget for text
+///   entry.
+/// * [KeyboardListener], a similar widget based on the newer [HardwareKeyboard]
+///   API.
```

Comment:
```suggestion
///  * [EditableText], which should be used instead of this widget for text
///    entry.
///  * [KeyboardListener], a similar widget based on the newer [HardwareKeyboard]
///    API.
```